### ISSUE TITLE
Edit Templates with data, after mod directory is created

### ISSUE THREAD

**Issue by KANAjetzt** [GodotModding/godot-mod-tool#44]
Create an easy way to make templates variable.

For *mod_main.gd*:

```gdscript
const MOD_DIR := "AuthorName-ModName" 
const MODNAME_LOG_NAME := "AuthorName-ModName
```

```gdscript
const {%MOD_NAME%}_MOD_DIR := "{%NAMESPACE%}-{%NAME%}" 
const {%MOD_NAME%}_LOG_NAME := "{%NAMESPACE%}-{%NAME%}"
```

**Comment by KANAjetzt** [comment]
I'm thinking that templates can include an `init_template.gd` script that gets executed by the Dev Tool if it exists.

**Comment by Qubus0** [comment]
yep that sounds good. i also figured out a way to stop godot from complaining. adding an empty file named `.gdignore` to a directory will exclude it from the whole editor. the downside would be that those templates can't be edited in the editor after that, but that should be fine since the templates A) won't be edited much and B) can just be edited in vsc and such

**Comment by KANAjetzt** [comment]
For reference 
- #148
